python/test/test_energy_storage_device_wrappers.py

```diff
@@ -59,6 +59,20 @@ def test_inspect_device(self):
             self.assertTrue(key in data)
         print(data)
 
+    def test_postprocessor_inspect(self):
+        ptree = PropertyTree()
+        ptree.parse_info('series_rc.info')
+        device = EnergyStorageDevice(ptree)
+        self.assertRaises(RuntimeError, device.inspect, 'postprocessor')
+
+        ptree = PropertyTree()
+        ptree.parse_info('super_capacitor.info')
+        ptree.put_int('dim', 3)
+        device = EnergyStorageDevice(ptree)
+        data = device.inspect('postprocessor')
+        self.assertTrue(isinstance(data, dict))
+        self.assertEqual(len(data), 0)
```

Comment:
You may use [os.path.isfile(path)](https://docs.python.org/3/library/os.path.html#os.path.isfile) to check that the file has been created.